srcpkgs/ly/files/ly/run

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+[ -r conf ] && . ./conf
+
+chvt "${TTY}"
```

Comment:
I don't love the idea of running `chvt`, but fair enough.

Reply:
I tested it without `chvt`, and unfortunately `ly` doesn't switch to its TTY on its own.